r_basics_visualize_data/r_basics_visualize_data.md

```diff
@@ -0,0 +1,1064 @@
+            <td style="padding: 0.5em;">
+            <p>27.0</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>109</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>140</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>26.0</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>39.5</p>
+            </td>
+        </tr><tr><td style="padding: 0.5em;">
+            <p>Depression with Anxiety (n=12m,8f)</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>130</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>145</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>25.0</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>19.8</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>103</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>142</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>24.9</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>40.1</p>
+            </td>
+        </tr><tr><td style="padding: 0.5em;">
+            <p>Controls / Neither Depression nor Anxiety (n=10m, 10f)</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>107</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>110</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>15.8</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>13.9</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>88</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>95</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>21.8</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>20.6</p>
+            </td>
+        </tr><tr><td style="padding: 0.5em;">
+            <p>Anxiety Alone (n=13m, 15f)</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>124</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>119</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>20.7</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>18.3</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>100</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>110</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>24.2</p>
+            </td>
+            <td style="padding: 0.5em;">
+            <p>20.5</p>
+            </td>
+        </tr></tbody></table>
+
+<div class = "question">
+Which of the following are ways to make this dataset "tidy"?  Check all that apply!
+
```

Comment:
this entire section is appearing all on one page. 

I was about to try to make some suggestions to hopefully fix it, but I'm realizing it's probably more efficient to have you do that on your end where you can see updates in realtime.